### PR TITLE
Added metapackage

### PR DESCRIPTION
**Pull request by ipa-rmb**

### CODE REVIEW

**Comment by diersma** [comment]
I'm happy to approve the pull if someone with ROS experience gives his ok @ipa-fmw / @ipa-josh?

File: sick_visionary_t/package.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<package format="2">
+
+  <name>sick_visionary_t</name>
+
+  <description>
+    Open source driver for the SICK Visionary-T 3D TOF camera.
+  </description>
+
+  <version>0.0.4</version>
+  <license>LGPL</license>
```
Comment (ipa-fmw) [inline sick_visionary_t/package.xml]: please use ```Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)``` (same as in https://github.com/SICKAG/sick_visionary_t/blob/indigo-devel/sick_visionary_t_driver/package.xml)

**Comment by ipa-josh** [comment]
license differes GPL vs. Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)

File: sick_visionary_t/package.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  <maintainer email="joshua.hampp@ipa.fraunhofer.de">Joshua Hampp</maintainer>
+  <maintainer email="marco.dierschke@sick.de">Marco Dierschke</maintainer>
+  
+  <url>http://wiki.ros.org/sick_visionary_t_driver</url>
```
Comment (ipa-fmw) [inline sick_visionary_t/package.xml]: please use http://wiki.ros.org/sick_visionary_t

**Comment by ipa-rmb** [comment]
Changed license and and url as requested.

**Pull request merged**